packages/framework/fluid-framework/api-report/fluid-framework.legacy.beta.api.md

```diff
@@ -4,6 +4,17 @@
 
 ```ts
 
+// @beta
```

Comment:
Note for reviewers: these additions are expected. `legacy` exports / reports now include non-legacy items of the same level or higher.